src/pages/q/[[...all]].tsx

```diff
@@ -211,6 +215,8 @@ export const getServerSideProps: GetServerSideProps = async function ({
     )
     try {
       initialInstructor = await loadInstructor(instructorSlug)
+
```

Comment:
here’s where I’d consider a single “load instructor” that loads from all sources